build.html

```diff
@@ -14,9 +14,9 @@
 <header>
   <nav>
     <ul>
-      <li><a href="#">Home</a></li>
+      <li><a href="index.html">Home</a></li>
```

Comment:
Please make sure the nav links work before submitting code. I fixed it on 3 files and will approve this PR.

Reply:
Thanks for the catch!